lib/cluster.rb

```diff
@@ -3,11 +3,15 @@
 
 class Cluster
   def initialize(config, tests)
-    @config = config
     @tests = tests
-    spin_up_nodes
+
+    print_foo()
   end
 
+
+  def print_foo():
+    print('foo')
```

Comment:
Hey, this is python. This is a ruby project

Reply:
Here's how to do that in ruby
```suggestion
  def print_foo
    puts 'foo'
  end
```